packages/forms-validators/src/validators/iban-validator.ts

```diff
@@ -14,7 +14,7 @@ export const IbanValidator = (props: BaseValidatorProps): null => {
         () => {
             return {
                 shouldValidate: value => {
-                    return value != null && value.trim.length === 0;
+                    return typeof value === "string" && value.trim().length > 0;
```

Comment:
You should only check for the value to be of string type. 
Trimming and checking for length is actual validation.

Reply:
Trimming and length checking is done to avoid validation when there is no value. Checking for IBAN should only occur when there is actual data, so that it could be optional field.

Reply:
Field with value `"    "` is not a valid IBAN.